src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -200,27 +211,16 @@ const HoloFuelDnaInterface = {
 
     // NOTE: Below we reflect our current change to the receive_payment API; the only param should now be the transaction's origin id
     accept: async (transactionId) => {
+      const transaction = await HoloFuelDnaInterface.transactions.getPending(transactionId)
       await createZomeCall('transactions/receive_payments_pending')({ origin: transactionId })
       return {
+        ...transaction,
```

Comment:
Similar to the 'decline' endpoint, let's just take the 'type', 'direction', and 'timestamp' from the values returned in 'transaction' object.

Reply:
👍

Reply:
actually, I think I should leave direction and type in here (but switch to using the timestamp from the fetched transaction as you suggest).

As I understand it, this endpoint specifically accepts an incoming offer, so we always know the type and direction. Having it be hardcoded here rather than implicit in the fetched transaction makes the code more self-documenting.

Reply:
Correct, for this endpoint, we will always now the type and direction.  I do actually like the idea of hard-coding this in for the purposes of readability and documentation. :100: Let's go with this approach.